ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/LdaptiveUserDao.java

```diff
@@ -170,6 +171,7 @@ private int enqueueAllByFilter(@Nonnull final Collection<User> users, @Nullable
             conn.open();
             SearchOperation search = new SearchOperation(conn);
             final SearchRequest request = new SearchRequest(this.baseSearchDn, filter);
+            request.setReturnAttributes("*", LDAP_ATTR_PASSWORDCHANGEDTIME);
```

Comment:
and pwdChangedTime is an operational attribute, which is why it wasn't included with User attributes